src/bartholomew.rs

```diff
@@ -50,6 +50,16 @@ pub fn render(req: Request) -> Result<Response> {
         config.base_url = Some(url);
     }
     eprintln!("Base URL: {:?}", &config.base_url);
+    eprintln!("Prepend route info : {:?}", &config.prepend_route_info);
+
+    if let Some(true) = config.prepend_route_info {
+        let route_info = match req.headers().get("spin-component-route") {
+            Some(route) => route.to_str().unwrap_or_default(),
```

Comment:
Is silently defaulting to not prefix correct here?  (I know it should be can't-happen.)

Reply:
Yes - that is to keep it a nonbreaking change.  As things stand today we parse only using `spin-path-info` - prefixing the route info has to be optional.